files/commonfiles/include/install.php

```
@@ -66,10 +59,17 @@ function xoops_module_install_modulebuilder(\XoopsModule $module)
     $helper->loadLanguage('common');
 
     //  ---  CREATE FOLDERS ---------------
+    $success = true;
     if ($configurator->uploadFolders && \is_array($configurator->uploadFolders)) {
         foreach (\array_keys($configurator->uploadFolders) as $i) {
-            $utility::createFolder($configurator->uploadFolders[$i]);
-            chmod($configurator->uploadFolders[$i], 0777);
+            $path = $configurator->uploadFolders[$i];
+            if (!\is_dir($path) && !$utility::createFolder($path)) {
```

PHPStan (2.1.46)

[error] Result of static method XoopsModules\Modulebuilder\Utility::createFolder() (void) is used.


(staticMethod.void)